src/test/kotlin/view/InputViewTest.kt

```diff
@@ -0,0 +1,24 @@
+package view
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.collections.shouldContainAll
+import java.io.ByteArrayInputStream
+
+private const val WHITE_SPACE = " "
+
+class InputViewTest : BehaviorSpec(
+    {
+        given("사용자의 입력을 받을 때") {
+            val inputView = InputView()
+            val strings = "12 32 2 1 4 5"
+            val expected = strings.split(WHITE_SPACE).map { it.toInt() }
```

Comment:
```suggestion
            val expected = strings
                .split(WHITE_SPACE)
                .map { it.toInt() }
```

```Kotlin
fun requestLottoNumber(): List<Int> {
        return readln()
            .split(WHITE_SPACE)
            .map { Number(it) }
            .map { it.getInt() }
    }
```
큰 건 아니고 소스 코드에서 줄바꿈을 이렇게 하셨길래 ㅎ_ㅎ suggestion으로 넣겠습니다!